tests/integration/kubernetes/k8s-confidential.bats

```diff
@@ -10,7 +10,9 @@ load "${BATS_TEST_DIRNAME}/confidential_common.sh"
 load "${BATS_TEST_DIRNAME}/tests_common.sh"
 
 setup() {
-	confidential_setup || skip "Test not supported for ${KATA_HYPERVISOR}."
+	if ! is_confidential_runtime_class; then
```

Comment:
Hi @stevenhorsman @fidencio !

`k8s-confidential.bats` should run only on confidential hardware, so we need something like:

`is_confidential_hardware || skip "Test is supported only on confidential hardware (which ${KATA_HYPERVISOR} is not)"`

Reply:
Updated!